CONDARC

```diff
@@ -0,0 +1,2 @@
+envs_dirs:
```

Comment:
I think this file needs to be named `.condarc` instead of `CONDARC`.

I'm also a little worried that when the volume mount for the user folders are run, it may null out anything in the folder anyways.

Reply:
I forgot to add the copy to that name. Can you edit the PR? I'm typing with only one hand today ;-p

Reply:
Ah no, you've got it. I try not to rename things during most Docker operations. 

Let me try it locally and see if it works as I think it should.

Reply:
Interesting, I do the opposite. I always go from a non-functional name to a functional one, to avoid messing up things locally in case I issue a command where the file lives.

Reply:
Ya, the `.condarc` gets hidden by the volume mount.

It might need to be placed somewhere else and then moved into place on startup to add to all the user folders.

Reply:
Ah a little further up that [page](https://zero-to-jupyterhub.readthedocs.io/en/latest/customizing/user-environment.html#about-user-storage-and-adding-files-to-it). I'm going to change that to copy to `/tmp` and then we can move it with a `postStart` command.

Reply:
Great! Thanks!